packages/o-spreadsheet-engine/src/types/commands.ts

```diff
@@ -221,6 +219,36 @@ export const readonlyAllowedCommands = new Set<CommandTypes>([
   "UPDATE_PIVOT",
 ]);
 
+export const lockedSheetAllowedCommands = new Set<Command["type"]>([
+  "LOCK_SHEET",
+  "UNLOCK_SHEET",
+  "UPDATE_LOCALE",
+  "MOVE_SHEET",
+  "DUPLICATE_SHEET",
+  "CREATE_SHEET",
+  "COPY",
+  "START",
+  "SCROLL_TO_CELL",
+  "ACTIVATE_SHEET",
+  "RESIZE_SHEETVIEW",
+  "SET_VIEWPORT_OFFSET",
+  "SET_FORMULA_VISIBILITY",
+  "SELECT_FIGURE", // not  sure
+  "EVALUATE_CHARTS",
+  "EVALUATE_CELLS",
+  "UNDO",
+  "REDO",
+  "REQUEST_UNDO",
+  "REQUEST_REDO",
+  "REPLACE_SEARCH",
```

Comment:
allowing replace on a locked sheet is strange. Does the command contains the sheetId if we replace on all sheet?